inference-engine/include/vpu/hddl_plugin_config.hpp

```diff
@@ -51,75 +52,102 @@ namespace InferenceEngine {
 namespace Metrics {
 
 /**
+* @deprecated  Use MYRIAD_METRIC_KEY(DEVICE_NUM) instead
 * @brief Metric to get a int of the device number, String value is METRIC_VPU_HDDL_DEVICE_NUM
 */
+INFERENCE_ENGINE_DEPRECATED("Use MYRIAD_METRIC_KEY(DEVICE_NUM) instead")
 DECLARE_VPU_HDDL_METRIC(DEVICE_NUM, int);
 
 /**
+* @deprecated  Use MYRIAD_METRIC_KEY(DEVICE_NAME) instead
 * @brief Metric to get a std::vector<std::string> of device names, String value is METRIC_VPU_HDDL_DEVICE_NAME
 */
+INFERENCE_ENGINE_DEPRECATED("Use MYRIAD_METRIC_KEY(DEVICE_NAME) instead")
 DECLARE_VPU_HDDL_METRIC(DEVICE_NAME, std::vector<std::string>);
 
 /**
+* @deprecated  Use MYRIAD_METRIC_KEY(DEVICE_THERMAL) instead
 * @brief  Metric to get a std::vector<float> of device thermal, String value is METRIC_VPU_HDDL_DEVICE_THERMAL
 */
+INFERENCE_ENGINE_DEPRECATED("Use MYRIAD_METRIC_KEY(DEVICE_THERMAL) instead")
 DECLARE_VPU_HDDL_METRIC(DEVICE_THERMAL, std::vector<float>);
 
 /**
+* @deprecated  Use MYRIAD_METRIC_KEY(DEVICE_ID) instead
 * @brief  Metric to get a std::vector<uint32> of device ids, String value is METRIC_VPU_HDDL_DEVICE_ID
 */
+INFERENCE_ENGINE_DEPRECATED("Use MYRIAD_METRIC_KEY(DEVICE_ID) instead")
```

Comment:
So you suggest using configuration keys without macros, but metrics with macros?

Reply:
Change to variables